blocks/api/paste.js

```diff
@@ -76,7 +78,9 @@ export function normaliseToBlockLevelNodes( nodes ) {
 }
 
 export default function( nodes ) {
-	return normaliseToBlockLevelNodes( nodes ).map( ( node ) => {
+	const prepare = compose( [ normaliseToBlockLevelNodes, gDocs, stripAttributes ] );
```

Comment:
@iseulde I removed the `{ test(), run() }` signature and made them simple functions. I figured that if there was a test to decide whether to run, the function can just do the test and return early.